server/validation/register.js

```diff
@@ -4,23 +4,13 @@ module.exports = function validateRegisterInput(data) {
   let errors = {};
   // Convert empty fields to an empty string so we can use validator functions
   data.name = !isEmpty(data.name) ? data.name : "";
-  data.email = !isEmpty(data.email) ? data.email : "";
   data.password = !isEmpty(data.password) ? data.password : "";
   // Name checks
   if (Validator.isEmpty(data.name)) {
     errors.name = "Name field is required";
   }
-  // Email checks
-  if (Validator.isEmpty(data.email)) {
-    errors.email = "Email field is required";
-  } else if (!Validator.isEmail(data.email)) {
-    errors.email = "Email is invalid";
-  }
   // Password checks
-  if (Validator.isEmpty(data.password)) {
-    errors.password = "Password field is required";
-  }
-  if (!Validator.isLength(data.password, { min: 6, max: 30 })) {
+  if (data.password && !Validator.isLength(data.password, { min: 6, max: 30 })) {
```

Comment:
have you been able to get this error to show up? I haven't, and I get a different error but the user creation still goes through 
![Screen Shot 2021-08-09 at 4 47 31 PM](https://user-images.githubusercontent.com/47487758/128788117-ac7d25bd-d38c-45c3-bcff-acb948b7893f.png)
![image](https://user-images.githubusercontent.com/47487758/128788135-7164a957-0aed-4908-a81a-1b110c1efdee.png)

Reply:
oh lol I forgot this existed, I just used my own validation instead (ofc it had bugs)

Reply:
there's client side validationa nd the server side one. What is displayed is the half implemented client one.